cms/static/sass/elements/_creative-commons.scss

```diff
@@ -1,10 +1,10 @@
 @font-face {
   font-family: 'CreativeCommons';
-  src: url('../fonts/CreativeCommons/cc.eot');
-  src: url('../fonts/CreativeCommons/cc.eot#iefix') format('embedded-opentype'),
-       url('../fonts/CreativeCommons/cc.woff') format('woff'),
-       url('../fonts/CreativeCommons/cc.ttf') format('truetype'),
-       url('../fonts/CreativeCommons/cc.svg#CreativeCommons') format('svg');
+  src: url($static-path + '/fonts/CreativeCommons/cc.eot');
```

Comment:
The $static-path variable is set in the theme to refer to the root of the edx-platform static assets.  This makes the sass files in edx-platform work to find their assets properly.  Themers don't need to copy assets to make it work.